src/components/CardQuirks.css

```
@@ -122,72 +144,106 @@
+  }
 }
-.Skatha:lang(es) .character-plays .text {
-  font-size: calc(6.6pt * 200/96);
-  letter-spacing: -0.5px;
+.Skatha:lang(es) .character-plays {
+  & .text {
+    font-size: calc(6.6pt * 200/96);
+  }
+  & .ColdSnap .text {
+    & ::first-line { letter-spacing: normal; }
+    letter-spacing: -0.5px;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)